src/index.css

```diff
@@ -72,3 +80,19 @@
     content: none !important;
   }
 }
+
+:global(.relative) {
```

Comment:
sure, but as they are very basic helper classes i guess this should be ok. but i'm open to critique.